test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests.Fakes/FakeExample.cs

```diff
@@ -13,6 +14,11 @@ public override IOpenApiExample<FakeClassModel> Build(NamingStrategy namingStrat
             this.Examples.Add(OpenApiExampleResolver.Resolve("second", "this is summary", new FakeClassModel() { Number = 3, Text = "Lorem Ipsum" }, namingStrategy));
             this.Examples.Add(OpenApiExampleResolver.Resolve("third", "this is summary", "this is description", new FakeClassModel() { Number = 2, Text = "Hello Ipsum" }, namingStrategy));
 
+            this.Examples.Add(OpenApiExampleResolver.Resolve("intValue", 1, namingStrategy));
+            this.Examples.Add(OpenApiExampleResolver.Resolve("stringValue", "stringValue", namingStrategy));
+            this.Examples.Add(OpenApiExampleResolver.Resolve("doubleValue", 0.123, namingStrategy));
+            this.Examples.Add(OpenApiExampleResolver.Resolve("date-timeValue", Convert.ToDateTime("2021.01.01"), namingStrategy));
+            this.Examples.Add(OpenApiExampleResolver.Resolve("booleanValue", false, namingStrategy));

```

Comment:
This `FakeExample` class is supposed for `FakeClassModel`, not your values. Please remove those lines.